### ISSUE TITLE
BridgeDb tool to create linksets in a Wizard mode for use by biologists

### ISSUE THREAD

**Issue by egonw** [bridgedb/BridgeDb#86]

**Comment by AlasdairGray** [comment]
@Chris-Evelo mentioned that this was similar to something that is developed for Goliath

**Comment by egonw** [comment]
No, it's different. What he told me is that Goliath can do the translation of identifiers to their identifiers.org URI representation.

**Comment by AlasdairGray** [comment]
Ah, may be we weren't quite talking about the same thing. But that would definitely be part of the solution anyway.
